kol/requirements.txt

```diff
@@ -11,6 +11,7 @@ requests==2.32.3
 scipy==1.13.1
 tqdm==4.66.4
 trimesh==4.4.1
+python-dotenv==1.0.1
```

Comment:
noooooo

also wait since when did we add all these random requirements, i think i need to clean some stuff up